ckanext/datagovuk/templates/organization/member_new.html

```diff
@@ -0,0 +1,90 @@
+  <h1 class="page-heading">
+    {% block page_heading %}{{ _('Edit Member') if user else _('Add Member') }}{% endblock %}
+  </h1>
+  {% block form %}
+  <form class="dataset-form form-horizontal add-member-form" method='post'>
+    <div class="control-set">
+      {% if not user %}
+        <label for="username">
+          {{ _('Existing User') }}
+        </label>
+        <span>
+          {{ _('If you wish to add an existing user, search for their username below.') }}
+        </span>
+      {% endif %}
+    </div>
+    <div class="control-set">
+      {% if user %}
+        <input type="hidden" name="username" value="{{ user.name }}" />
+        <input id="username" name="username" type="text" value="{{ user.name }}"
+        disabled="True" class="control-medium">
+      {% else %}
+         <input id="username" type="text" name="username" placeholder="{{ _('Username') }}"
+        value="" class="control-medium" data-module="autocomplete"
+        data-module-source="/api/2/util/user/autocomplete?q=?">
+      {% endif %}
+      {% if not user %}
+        <label for="email">
+          {{ _('New User') }}
+        </label>
+        <p>
+          {{ _('If you wish to invite a new user, enter their email address.') }}
+        </p>
+        <div>
+          <input id="email" type="email" name="email" placeholder="{{ _('Email address') }}" >
+        </div>
+      {% endif %}
+    </div>
+
+    <div class="control-set">
+      <label for="role">
+        {{ _('Role') }}
+      </label>
+      <select name="role">
+        {% for role in h.roles() %}
+          <option value="{{ role.lower() }}" {% if c.user_role == role.lower() %}selected="selected"{% endif %}>{{ role }}</option>
+        {% endfor %}
+      </select>
+      <div class="form-actions">
+        {% if user %}
+          <a href="{% url_for controller='organization', action='member_delete', id=c.group_dict.id, user=user.id %}" class="btn btn-danger pull-left" data-module="confirm-action" data-module-content="{{ _('Are you sure you want to delete this member?') }}">{{ _('Delete') }}</a>
+          <button class="btn btn-primary" type="submit" name="submit" >
+            {{ _('Update Member') }}
+          </button>
+        {% else %}
+          <button class="btn btn-primary" type="submit" name="submit" >
+            {{ _('Add Member') }}
+          </button>
+        {% endif %}
+      </div>
+    </div>
+  </form>
+  {% endblock %}
+{% endblock %}
+
+{% block secondary_content %}
+{{ super() }}
+<div class="module module-narrow module-shallow">
+  <h2 class="module-heading">
+    <i class="fa fa-lg fa-info-circle"></i>
+    {{ _('What are roles?') }}
+  </h2>
+  <div class="module-content">
+    {% trans %}
+      <p><strong>Admin:</strong> Can add/edit and delete datasets, as well as
+        manage organization members.</p>
```

Comment:
Should this (and below) say "publisher members"?

Reply:
Fixed